### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by mustabbas**
### Change Done 
- [ ] Implemented a basic UI with plain HTML:
- [ ] Created a collection that keeps a list of books.
- [ ] Created a function to add a new book to the collection, with title and author.
- [ ] Created a function to remove a book from the collection.
- [ ] Displayed all books saved in the collection in the top part of the page.
- [ ] Added data in the browser's memory by using localStorage.

### File Changed
- [ ] index.html
- [ ] script.js

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:48]: - Kindly fix your linter errors in your pull request.
- Provide general information to your README File.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+function removeBookList() {
+  for (let i = 0; i < BookArray.length; i += 1) {
+    if (document.getElementById(BookArray[i].id) != null) {
+      // eslint-disable-next-line no-loop-func
+      document.getElementById(BookArray[i].id).addEventListener('click', () => {
+        function isId(value) {
+          return value.id !== BookArray[i].id;
+        }
+        document.getElementById(`title${i}`).remove();
```
Comment (carlos-ssh) [inline script.js]: **Kindly fix this error:**
- When i try to remove an item from the list. This error is displayed on the chrome console:
Screen Shot 2021-09-13 at 19 33 29

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mustabbas , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+}
+
+function removeBookList() {
+  for (let i = 0; i < BookArray.length; i += 1) {
+    if (document.getElementById(BookArray[i].id) != null) {
+      // eslint-disable-next-line no-loop-func
+      document.getElementById(BookArray[i].id).addEventListener('click', () => {
+        function isId(value) {
+          return value.id !== BookArray[i].id;
```
Comment (carlos-ssh) [inline script.js]: - **This issue stills displaying in console:**
Screen Shot 2021-09-13 at 19 53 46

- **Update your README file.**

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mustabbas , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Hi @mustabbas and @Cathella, 

### Status: Approved ✔️

:+1:  You have done well implementing the suggested changes.

Your project is complete!  There are some optional changes I've suggested as **I think it was not mentioned to you in the previous review to complete all the requirements** Please fix that before merging, other than that nothing else to say.

Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[**Highly recommended**]
- The browser after refresh doesn't show any data, Please make to show the book which is stored in the localstorage and to push the next item in the same array as well.

Screenshot from 2021-09-14 15-28-01



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**